cw-orch-daemon/src/sync/builder.rs

```diff
@@ -80,7 +99,8 @@ impl DaemonBuilder {
 
     /// Build a Daemon
     pub fn build(&self) -> Result<Daemon, DaemonError> {
-        let rt_handle = RUNTIME.handle().clone();
+        let rt_handle = self.handle.clone().unwrap_or(RUNTIME.handle().clone());
+
```

Comment:
Shouldn't it be `unwrap_or_else` so we don't spawn runtime, when only custom used?

Reply:
Oh, I thought lazy_static code only executed when used, sorry